src/platform/nrfconnect/FactoryDataProvider.cpp

```diff
@@ -129,6 +133,17 @@ CHIP_ERROR FactoryDataProvider<FlashFactoryData>::MoveDACPrivateKeyToSecureStora
     uint8_t clearedDACPrivKey[kDACPrivateKeyLength];
     memset(clearedDACPrivKey, 0x00, sizeof(clearedDACPrivKey));
 
+// If key should be migrated to KMU save the KMU key slot to keyId.
+#ifdef CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU
+    mDACPrivKeyId = static_cast<psa_key_id_t>(PSA_KEY_HANDLE_FROM_CRACEN_KMU_SLOT(
+#ifdef CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_ENCRYPTED
+        CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED,
```

Comment:
What are the pros of using raw scheme? I'm wondering should we support both options, or just encrypt it always.

Reply:
the purpose is to have only 2 slots instead of 4. If someone wants to save the space in KMU storage for other keys it may be profitable, but a bit less secure. Bye default we use encryption.